shibboleth/resources/prod/proxy.conf

```diff
@@ -0,0 +1,27 @@
+# Route requests to Spring Boot application EFT via AJP
+<VirtualHost edview.unilu.ch:443>
+    ProxyPreserveHost On
+    ProxyRequests Off
+    ProxyIOBufferSize 65536
+    RedirectMatch ^/$ /edview
+
+    ProxyPass /eft ajp://edview.unilu.ch:8448/eft
```

Comment:
I think this is an outdated version of the proxy. Have you pulled my changes?

I have already adjusted this to match the test environment.